private/getVariableDeclarationIdentifierNames.js

```diff
@@ -15,7 +15,32 @@ module.exports = function getVariableDeclarationIdentifierNames(
 ) {
   const names = [];
 
-  // Recursive function and call to go here…
+  /**
+   * Recursively collects identifier names from variable declarations.
+   * @kind function
+   * @name collectIdentifierNames
+   * @param {object} node A Babel AST Node.
+   * @ignore
+   */
+  function collectIdentifierNames(node) {
+    if (node.type === 'Identifier') names.push(node.name);
```

Comment:
Can we please do this as a `switch` on `node.type` instead of a series of `if` statements?